README.md

```diff
@@ -1,13 +1,400 @@
-wordpress-docker
-============
+# <a name="about"></a>About
 
-Dockerfile source for WordPress [docker](https://docker.io) image.
+This image contains an installation of WordPress 5.x served by an Apache HTTP
+Server on a PHP 7.x runtime.
 
-# Upstream
+For more information, see the
+[Official Image Launcher Page](https://console.cloud.google.com/launcher/details/google/wordpress5-php7-apache).
 
-This source repo was originally copied from:
-https://github.com/docker-library/wordpress
+Pull command (first install [gcloud](https://cloud.google.com/sdk/downloads)):
 
-# Disclaimer
+```shell
+gcloud docker -- pull marketplace.gcr.io/google/wordpress5-php7-apache
+```
 
-This is not an official Google product.
+Dockerfile for this image can be found [here](https://github.com/GoogleCloudPlatform/wordpress-docker/tree/master/5/php7/debian9/apache).
```

Comment:
Remove this link, it's redundant in this case

Reply:
Done